src/main/scala/net/bdew/ae2stuff/machines/wireless/BlockWireless.scala

```diff
@@ -49,8 +51,18 @@ object BlockWireless
   ): util.ArrayList[ItemStack] = {
     val stack = new ItemStack(this)
     val te = world.getTileEntity(x, y, z).asInstanceOf[TileWireless]
-    if (te != null && te.color != AEColor.Transparent) {
-      stack.setItemDamage(te.color.ordinal() + 1)
+    if (te != null) {
+      if (te.isHub) {
+        if (te.color != AEColor.Transparent) {
+          stack.setItemDamage(te.color.ordinal() + 18)
```

Comment:
Some of these magic numbers could use constants

Reply:
Because hub using second layer of wireless, need use some offsets for convert color<->meta, aslo AEColor store transparent as last index unlike items meta which use first index. There no life without some magic.